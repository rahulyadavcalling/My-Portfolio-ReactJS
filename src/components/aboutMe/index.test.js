import React from "react";
import renderer from "react-test-renderer";

import AboutMe from "./index.js";

it("renders correctly", () => {
  const tree = renderer.create(<AboutMe />).toJSON();
  expect(tree).toMatchSnapshot();
});
