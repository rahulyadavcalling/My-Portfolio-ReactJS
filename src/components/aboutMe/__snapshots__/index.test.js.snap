// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa iDooKw"
>
  <div
    className="sc-bwzfXH gjBHOH"
  >
    <p>
      Hey, My name is Reyansh Mishra, I am an Android and iOS Developer, previously I was working with Tacto Infomedia Pvt. Ltd. currently, I am freelancing. In my college days, I created an Android App called Boom because of which Ianded a job in my first company that is Tacto.
      <br />
      <br />
       When I was in final year of my graduation I had PPO from L&T Infotech and Infosys but I chose to work with a Startup because learning new skills and development work excites me more than working in an MNC. I am an Android enthusiast and have created a couple of open source projects on my GitHub profile.
      <br />
      <br />
       When I was in final year of my graduation I had PPO from L&T Infotech and Infosys but I chose to work with a Startup because learning new skills and development work excites me more than working in an MNC. I am an Android enthusiast and have created a couple of open source projects on my GitHub profile.
      <br />
      <br />
       I started learning android when I had to create a project for my graduation and I learned android development from youtube and with the help of blogs and open source projects.
      <br />
      <br />
       My biggest inspiration is "Mr. Elon Musk" maybe I will get lucky someday and meet him.
      <br />
      <br />
       In my spare time, I either play a video game on my PlayStation 4 or use the internet. I donâ€™t like to study books so I use Quora to gain a little bit of knowledge, sometimes I write there but not much.
    </p>
  </div>
</div>
`;
